charts/portworx/templates/storage-cluster.yaml

```diff
@@ -228,18 +228,27 @@ spec:
     value: portworx
 
   stork:
-    {{- if (and (.Values.stork) (eq .Values.stork true))}}
+    {{- if (and (.Values.stork.enabled) (eq .Values.stork.enabled true))}}
     enabled: true
-    {{- if .Values.storkVersion }}
-    image: {{ template "px.getStorkImage" . }}:{{ required "A valid Image tag is required in the SemVer format" .Values.storkVersion }}
+    {{- if .Values.stork.storkVersion }}
+    image: {{ template "px.getStorkImage" . }}:{{ required "A valid Image tag is required in the SemVer format" .Values.stork.storkVersion }}
     {{- end }}
-    {{- with .Values.storkSpec.volumes }}
+
+    {{- if (and .Values.stork.lockImage (eq .Values.stork.lockImage true)) }}
+    lockImage: true
```

Comment:
let's remove this since lockImage is not being used anymore

Reply:
Removed